verification/src/transaction_verifier.rs

```diff
@@ -358,6 +371,21 @@ impl<DL: CellDataProvider + HeaderProvider + ExtensionProvider + Send + Sync + C
         Ok(ret)
     }
 
+    /// Perform resumable script verification with signal
+    /// The verification will be interrupted when receiving a signal,
+    /// it will hang until the `Resume` command is received.
+    pub async fn resumable_verify_with_signal(
```

Comment:
Is this method exclusively used for tests?

Reply:
no, this will be called in hot path.